src/Infrastructure/Adapter/Storage.php

```diff
@@ -36,8 +36,26 @@ public function delete(string $path): void
         $this->storage->delete($path);
     }
 
-    public function get(string $path): string
+    public function getUrl(string $path): string
     {
         return \sprintf('%s/%s', $this->mediaLocation, $path);
     }
+
+    public function read(string $path): ?string
+    {
+        if (!$this->storage->has($path)) {
+            return null;
+        }
+
+        return base64_encode($this->storage->read($path));
```

Comment:
Il vaut peut-être mieux faire le `base64_encode` en dehors (on peut avoir besoin de lire un fichier sans l'encoder en base64)

```suggestion
        return $this->storage->read($path);
```